testing/MLDB-1305_rowNames_join.py

```diff
@@ -26,6 +26,83 @@
 dataset.record_row("row2", [["col2", "b", now], ["otherRow", "row2", now]])
 dataset.commit()
 
+#MLDB-1454 rownames in WHERE regular
+
+mldb.log("MLDB-1454 rownames in WHERE regular")
+
+expected = [["_rowName","col1"],["row1","a"]]
+
+res = mldb.perform("GET", "/v1/query", [["q", """
+    SELECT *
+    FROM dataset1
+    WHERE dataset1.col1 = 'a'
+    """],['format', 'table']])
+
+assert res["statusCode"] != 404
```

Comment:
Why don't you use python unittest?  I thought we would be adopting it for new tests.